README.md

```diff
@@ -1,10 +1,24 @@
 # AI studio
 
+## Installing a released version
+
+You can install this extension using this [link](podman-desktop:extension/redhat.ai-studio).
```

Comment:
FYI this link is not being rendered ( GitHub is not rendering links that are not with http or https) so I'm not sure you want to display it like this ?